### PR TITLE
.editorconfig and CPP style-guides improvements

### PR DESCRIPTION
**Pull request by Bravo555**

### CODE REVIEW

File: cpp.draft.md [hunk 1]
```diff
@@ -1,7 +1,60 @@
...
+```
+foo_bar.cpp
+```
+
+##Formatting
+###Spaces vs. Tabs
+Tabs for indentation (4 spaces width), spaces for aligning.
+
+```c++
+if (q || p) {
```
Comment (Bravo555) [inline cpp.draft.md:57]: Are we sure we want these codeblock openings instead of:

``` c++
if(q || p)
{

}
```

**Review by Bravo555 — COMMENTED** [review]
One thing:

**Comment by Bravo555** [comment]
And Wojtek, you can revoke this write access you gave us.

**Comment by Bravo555** [comment]
And another thing: i think i'll be better to configure master branches to require code reviews before merging.

**Review by JJendryka — APPROVED** [review]

File: cpp.draft.md (hunk 1 above)
Comment (irth) [reply cpp.draft.md:57]: Yep. 'coz I said so.

**Pull request merged**